scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/example/logisticregression/VflLogisticRegression.scala

```diff
@@ -100,7 +70,6 @@ object VflLogisticRegression {
       opt[String]('r', "rowKeyName")
         .text("row key name of data")
         .action((x, params) => params.copy(rowKeyName = x))
-        .required()
       opt[String]('l', "learningRate")
         .text("learning rate of training")
         .action((x, params) => params.copy(learningRate = x.toFloat))
```

Comment:
change below to `FlContext.initFlContext`

Reply:
This is in #3687